src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  let res = 0;
+
+  if (days >= 3 && days < 7) {
+    res = cost * days - 20;
```

Comment:
```suggestion
    res = cost * days - 20;
```
Вираз `cost * days` в тебе повторюється тричі. Давай створимо змінну для нього